cmd/kubeadm/app/apis/kubeadm/types.go

```diff
@@ -69,6 +70,13 @@ type API struct {
 	BindPort int32
 }
 
+// kube-proxy configuration
+type KubeProxy struct {
```

Comment:
We're planning to source the full ComponentConfig struct for this: https://github.com/kubernetes/kubernetes/pull/53645, and upload that to the ConfigMap

Reply:
or wait; I meant that `MasterConfiguration.KubeProxy.Config` will point to the `kubeproxyconfig.k8s.io/KubeProxyConfiguration`